community-operators/hazelcast-enterprise/hazelcast-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,171 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    capabilities: Basic Install
+    alm-examples: |-
+      [{
+       "apiVersion": "hazelcast.com/v1alpha1",
+       "kind": "Hazelcast",
+       "metadata": {
+          "name": "hz"
+       },
+       "spec": {
+          "image": {
+            "tag": "3.11.2"
+          },
+          "hazelcast": {
+            "licenseKeySecretName": "hz-license-key"
+          },
+          "securityContext": {
+            "runAsUser": 65534,
+            "fsGroup": 65534
+          }
+       }
+      }]
+    categories: "Database"
+    certified: false
+    containerImage: hazelcast/hazelcast-enterprise-operator:0.0.1
+    createdAt: 2019-03-14T09:58:52Z
+    description: Install Hazelcast Enterprise cluster.
+    support: Hazelcast, Inc
+  name: hazelcast-operator.v0.0.1
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: Hazelcast Enterprise cluster.
+      displayName: Hazelcast Enterprise
+      group: hazelcast.com
+      kind: Hazelcast
+      name: hazelcasts.hazelcast.com
+      version: v1alpha1
+  description: |
+    Hazelcast IMDG Enterprise is the most widely used in-memory data grid with hundreds of thousands of installed clusters around the world. It offers caching solutions ensuring that data is in the right place when it’s needed for optimal performance.
+
+    ### Before Your Start
+
+    Add a Secret within the Project that contains the Hazelcast License Key. If you don't have one, get a trial key from this [link](https://hazelcast.com/hazelcast-enterprise-download/trial/).
+
+        $ oc create secret generic hz-license-key --from-literal=key=LICENSE-KEY-HERE
```

Comment:
`community-operators` are dedicated to OpenShift, so it's better to use `oc`. Otherwise we would need to ask users to configure their `kubectl` to use OpenShift cluster (which is not automatic I guess). You `kubectl` in `upstream-community-operators` (which is dedicated to k8).

Reply:
ok